.github/workflows/weekly.yml

```diff
@@ -30,34 +30,21 @@ jobs:
     strategy:
       matrix:
         flink_branches: [{
-          flink: 1.18-SNAPSHOT,
-          branch: main
-        }, {
           flink: 1.19-SNAPSHOT,
-          jdk: '8, 11, 17, 21',
```

Comment:
Why is it okay to remove the JDK matrix?

Reply:
It's defined a few lines below. We previously had different JDK versions for Flink releases, but that's not needed anymore since 1.19 and 1.20 have the same supported JDK versions.